gdal/swig/python/gdal-utils/osgeo_utils/auxiliary/util.py

```diff
@@ -361,3 +374,25 @@ def get_scales_and_offsets(bands: MaybeSequence[gdal.Band]) -> Tuple[bool, Maybe
         scales, offsets = scales[0], offsets[0]
     return is_scaled, scales, offsets
 
+
+def set_scales_and_offsets(bands: Union[PathOrDS, MaybeSequence[gdal.Band]],
```

Comment:
this method is not used by this PR

Reply:
Right, I think I found another bug in related to `gdallocationinfo` and `scale` but I wanted to dig deeper in a new PR.
Would you suggest I'll remove it for now?

Reply:
> Would you suggest I'll remove it for now?

yes please

Reply:
done